integration_tests/models/sql/sample.sql

```diff
@@ -0,0 +1 @@
+SELECT 1 as sample_column
```

Comment:
I think the tests are failing because `sample` is a reserved work on Snowflake. Can you call this model something different (and change any downstream references)? I can kick off the tests again once that's done